tests/backend/user/test_apiv2_human_find.py

```diff
@@ -8,6 +8,71 @@
 from tests.backend.common import human_find
 
 
+class NonDeterministicOrderedResults(list):
```

Comment:
Would be better to keep this here and have `test_apiv1_human_find.py` import it from there instead of duplicate the code

Reply:
done